schemas/JSON/manifests/v1.1.0/manifest.installer.1.1.0.json

```diff
@@ -275,6 +277,115 @@
       "uniqueItems": true,
       "description": "List of appx or msix installer restricted capabilities"
     },
+    "Market": {
+      "type": "string",
+      "pattern": "^[A-Z]{2}$",
+      "description": "The installer target market"
+    },
+    "MarketArray": {
+      "type": [ "array", "null" ],
+      "uniqueItems": true,
+      "maxItems": 256,
+      "items": {
+        "$ref": "#/definitions/Market"
+      },
+      "description": "Array of markets"
+    },
+    "Markets": {
+      "description": "The installer markets",
+      "type": [ "object", "null" ],
+      "oneOf": [
+        {
+          "properties": {
+            "AllowedMarkets": {
+              "$ref": "#/definitions/MarketArray"
+            }
+          },
+          "required": [ "AllowedMarkets" ]
+        },
+        {
+          "properties": {
+            "ExcludedMarkets": {
+              "$ref": "#/definitions/MarketArray"
+            }
+          },
+          "required": [ "ExcludedMarkets" ]
+        }
+      ]
+    },
+    "InstallerAbortsTerminal": {
+      "type": [ "boolean", "null" ],
+      "description": "Indicates whether the installer will abort terminal. Default is false"
+    },
+    "ReleaseDate": {
+      "type": [ "string", "null" ],
+      "format": "date",
+      "description": "The installer release date"
+    },
+    "InstallLocationRequired": {
+      "type": [ "boolean", "null" ],
+      "description": "Indicates whether the installer requires an install location provided"
+    },
+    "RequireExplicitUpgrade": {

```

Comment:
>RequireExplicitUpgrade [](http://example.com/codeflow?start=5&length=22)

Also add RequireElevation field

Reply:
I think we need an enum for that, but I cannot remember exactly the reasoning for that.  But at least at a minimum there are both elevation required installers and elevation rejecting installers.  So it would be better to have an enum value that we can set `ElevationBehavior` or something like that.

Reply:
More thoughts on values for it:
```
ElevationRequired // intended to indicate that the caller must be elevated because the installer will not elevate itself
ElevatesSelf
RestrictsSelf // Better word than restrict?
RestrictedRequired
```
Then we could add more as we find need of them.  For instance `ElevationRequiredForSilent` was a problem for MSI before we fixed it, but I know that for instance WebView2 falls into this category currently.  It self elevates for its normal unattended pattern, but does not for a silent install (basically same as msiexec behavior).

Reply:
Per offline chat, we'll do an enum of elevationRequired, elevationProhibited and elevatesSelf